README.md

```diff
@@ -1,57 +1,202 @@
-3. ...
+### Prerequisites
 
++ [Python 3.11](https://www.python.org/downloads/) 
++ [Azure Functions Core Tools](https://learn.microsoft.com/azure/azure-functions/functions-run-local?tabs=v4%2Cmacos%2Ccsharp%2Cportal%2Cbash#install-the-azure-functions-core-tools)
++ Install [Docker](https://www.docker.com/)
++ Install [Azurite storage emulator](https://learn.microsoft.com/azure/storage/common/storage-use-azurite). 
++ Clone the repo
 
-## Demo
+### Deploy language models
 
-A demo app is included to show how to use the project.
+1. [Create an Azure subscription with a valid payment method](https://azure.microsoft.com/pricing/purchase-options/pay-as-you-go). Free or trial Azure subscriptions *won't* work. 
 
-To run the demo, follow these steps:
+1. [Create a project in Azure AI Foundry](https://learn.microsoft.com/azure/ai-studio/how-to/create-projects?tabs=ai-studio)
 
-(Add steps to start up the demo)
+1. Go to **Model catalog** on the left menu and search for the following models to deploy:
+    - GPT-4
+    - GPT-3.5-turbo
+    - GPT-4o-mini
+    - Phi-4 ([small language model by Microsoft](https://techcommunity.microsoft.com/blog/aiplatformblog/introducing-phi-4-microsoft%E2%80%99s-newest-small-language-model-specializing-in-comple/4357090) that has advanced reasoning capabilities in areas like math and science)
+
+    ![Screenshot of model catelog on Azure AI Foundry](./media/model-catalog.png)
+  
+
+### Get endpoints and keys for models
+
+You'll need the model API key and endpoint for the next step.
+
+Go to the **Overview** tab of the project where models are deployed. API key is on the top.
+
+To get the endpoint, click on **Azure AI inference** under "Included capabilities":
+
+  ![Connection info](media/connection-info.png)
+
+### Set up Durable Task Scheduler emulator 
+
+1. Pull Docker image:
+    ```bash
+    docker pull mcr.microsoft.com/dts/dts-emulator:v0.0.5
+    ```
+1. Run Docker image:
+    ```bash
+    docker run -itP mcr.microsoft.com/dts/dts-emulator:v0.0.5
+    ```
+
+  The emulator exposes several ports: 
+  - `8080`: gRPC endpoint that allows the app to connect to the scheduler
+  - `8082`: endpoint for monitoring dashboard  
+
+   ![Docker desktop view of emulator container](./media/emulator-container.png)
+
+   Use the port number mapped to `8080` in your durable task scheduler connection string. For example, in this case, the connection string would be `Endpoint=http://localhost:55000>;Authentication=None`
+
+### Run app using Visual Studio Code
+
+1. Open **app** folder in a new terminal 
+2. Open VS Code by entering `code .` in the terminal
+3. In the root folder, create a file named `local.settings.json` with the following, filling in connection information from the previous step: 
+    ```json
+    {
+      "IsEncrypted": false,
+      "Values": {
+          "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+          "MODEL_ENDPOINT": "https://<resource name>.services.ai.azure.com/models",
```

Comment:
I would rename this this `MODELS_ENDPOINT`. Using the singular form is confusing because it implies that this environment variable is only used for a single model.